app.js

```diff
@@ -1,14 +1,34 @@
 const express = require('express');
-const { getTopics } = require('./controllers/topicsController')
+const { getTopics, getArticlesById } = require('./controllers/controllers')
 console.log(getTopics)
 const app = express();
 
-app.get('/api/topics', getTopics)
+app.get('/api/topics', getTopics);
+app.get('/api/articles/:article_id', getArticlesById);
```

Comment:
I might suggest `getArticleById` (singular) would be a more appropriate name for this controller

Reply:
Done! Thanks :)

Reply:
also changed fetcharticles to fetcharticle